### ISSUE TITLE
Results from the original paper

### ISSUE THREAD

**Issue by WalkerJG** [Wang-Shuo/GraphRec_PyTorch#2]
Hi, have you compared the results of your implementation and those reported in the original paper? Is there any difference?

**Comment by Wang-Shuo** [comment]
For the Ciao dataset, I got MAE of 0.7953 and RMSE of 1.0944, which are both worse than the original claims.  I used 80% of the data as a training set and trained the model for 30 epochs at a learning rate 0.001. The batch size is 256 and the embedding  size is 64.

**Comment by iYiYaHa** [comment]
Well... I have also tried to rerun their own implementation, but it takes unbearable time to train the model.
